<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsPointDisplacementRendererWidgetBase</class>
 <widget class="QWidget" name="QgsPointDisplacementRendererWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="mCenterSymbolLabel">
       <property name="text">
        <string>Center symbol:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCenterSymbolPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="mRendererLabel">
       <property name="text">
        <string>Renderer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mRendererComboBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="mRendererSettingsButton">
     <property name="text">
      <string>Renderer settings...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="mDisplacementCirclesGroupBox">
     <property name="title">
      <string>Displacement circles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="mCircleWidthLabel">
          <property name="text">
           <string>Circle pen width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="mCircleWidthSpinBox"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="mCircleColorLabel">
          <property name="text">
           <string>Circle color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsColorButton" name="mCircleColorButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="mCircleRadiusLabel">
          <property name="text">
           <string>Circle radius modification:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="mCircleModificationSpinBox">
          <property name="minimum">
           <double>-99.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="mDistanceToleranceLabel">
          <property name="text">
           <string>Point distance tolerance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="mDistanceSpinBox">
          <property name="decimals">
           <number>7</number>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="mLabellingGroupBox">
     <property name="title">
      <string>Labels</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="mLabelAttributeLabel">
          <property name="text">
           <string>Label attribute:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mLabelFieldComboBox"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="mLabelFontButton">
          <property name="text">
           <string>Label font...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>94</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="mLabelColorLabel">
          <property name="text">
           <string>Label color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsColorButton" name="mLabelColorButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="mScaleDependentLabelsCheckBox">
        <property name="text">
         <string>Use scale dependent labelling</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="mMaxScaleLabel">
          <property name="text">
           <string>max scale denominator:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mMaxScaleDenominatorEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
